### PR TITLE
workflow: cucushift-installer-rehearse-azure-ipi-ovn-hypershift

### PR DESCRIPTION
**Pull request by LiangquanLi930**
add workflow: `cucushift-installer-rehearse-azure-ipi-ovn-hypershift`

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
@jianlinliu @liangxia could you help to review this PR? This failed is a known cause, https://issues.redhat.com/browse/OCPQE-12617

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -904,6 +904,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-upi-proxy
+- as: e2e-azure-ipi-ovn-hypershift-mgmt-p2
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:907]: Sort the job in alphanumeric order

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml]: done

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/hypershift/deprovision/cucushift-installer-rehearse-azure-ipi-ovn-hypershift-deprovision-chain.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-ovn-hypershift-deprovision
+  steps:
+  - chain: hypershift-azure-destroy
+  - chain: cucushift-installer-rehearse-azure-ipi-deprovision
+  env:
+    - name: HYPERSHIFT_BASE_DOMAIN
+      default: "qe.azure.devcluster.openshift.com"
+    - name: HYPERSHIFT_AWS_REGION
```
Comment (heliubj18) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/hypershift/deprovision/cucushift-installer-rehearse-azure-ipi-ovn-hypershift-deprovision-chain.yaml:9]: aws region ?  change to azure....

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/hypershift/provision/cucushift-installer-rehearse-azure-ipi-ovn-hypershift-provision-chain.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  as: cucushift-installer-rehearse-azure-ipi-ovn-hypershift-provision
+  steps:
+  - chain: cucushift-installer-rehearse-azure-ipi-ovn-provision
+  - chain: hypershift-install
+  - chain: hypershift-azure-create
+  env:
+  - name: HYPERSHIFT_BASE_DOMAIN
+    default: "qe.azure.devcluster.openshift.com"
+  - name: HYPERSHIFT_AWS_REGION
+    default: "us-east-2"
```
Comment (heliubj18) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/hypershift/provision/cucushift-installer-rehearse-azure-ipi-ovn-hypershift-provision-chain.yaml:11]: region is incorrect

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/hypershift/deprovision/cucushift-installer-rehearse-azure-ipi-ovn-hypershift-deprovision-chain.yaml (hunk 2 above)
Comment (LiangquanLi930) [reply ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/hypershift/deprovision/cucushift-installer-rehearse-azure-ipi-ovn-hypershift-deprovision-chain.yaml:9]: https://github.com/openshift/release/pull/33460 wait for merge

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/hypershift/provision/cucushift-installer-rehearse-azure-ipi-ovn-hypershift-provision-chain.yaml (hunk 3 above)
Comment (LiangquanLi930) [reply ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/hypershift/provision/cucushift-installer-rehearse-azure-ipi-ovn-hypershift-provision-chain.yaml:11]: https://github.com/openshift/release/pull/33460 wait for merge

**Comment by LiangquanLi930** [comment]
@heliubj18 @liangxia @jianlinliu  PTAL again, thanks

**Comment by heliubj18** [comment]
LGTM

**Pull request merged**